_includes/cloud-db/cloud-summary-db-version.md

```diff
@@ -0,0 +1,2 @@
+Each database is displayed with a version - this represents the version of Featurebase that the database is running. 
```

Comment:
This is great and thanks for taking advantage of the includes :)

Two things to point out:
* the first sentence is the one that's the most important and hasn't been mentioned anywhere else.
* the second sentence is a duplication that's also found at the bottom of the `cloud-db-versions.md` page, so can be removed.